daemon/graphdriver/aufs/aufs.go

```diff
@@ -1,16 +1,18 @@
+// +build !windows
+
 /*
 
 aufs driver directory structure
 
 .
 ├── layers // Metadata of layers
-│   ├── 1
```

Comment:
No it's actually a non-breaking space `\xc2\xa0` instead of a regular space. http://www.fileformat.info/info/unicode/char/00a0/index.htm which isn't a big change.

A fix that could happen in this or in another PR is to remove the newline between the comment and the package declaration to make it a doc comment. In which case it'd be great to prepend the directory structure illustration with one or two spaces so that `godoc` renders it correctly.